metricbeat/module/jolokia/jmx/config.go

```diff
@@ -482,7 +482,7 @@ func (pc *JolokiaHTTPPostFetcher) Fetch(m *MetricSet) ([]common.MapStr, error) {
 	if logp.IsDebug(metricsetName) {
 		for _, r := range httpReqs {
 			m.log.Debugw("Jolokia request URI and body",
-				"httpMethod", r.HTTPMethod, "URI", r.URI, "body", string(r.Body), "type", "request")
+				"httpMethod", r.HTTPMethod, "URI", m.BaseMetricSet.HostData().SanitizedURI+r.URI, "body", string(r.Body), "type", "request")
```

Comment:
Some lines below `m.http.GetURI()` is used to print this URI, should we use the same here?
```suggestion
				"httpMethod", r.HTTPMethod, "URI", m.http.GetURI(), "body", string(r.Body), "type", "request")
```

They probably produce the same result, but we may wonder in the future why this is different.

Reply:
Thanks Jaime. Updated to the simplified m.http.GetURI call for now.